.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -95,6 +95,30 @@ steps:
             - exit_status: '*'
               limit: 1
 
+      - command: .buildkite/scripts/steps/functional/security_serverless_rule_management.sh
```

Comment:
There's a flag called `SKIP_CYPRESS` in this pipeline - could you please make these added steps respect it? (example below these)

I think some usages of this pipeline won't want to care about cypress tests, but a reduced set of functional testing only.

Reply:
Absolutely!! Thanks!! :)